src/assets/css/prism-base16-monokai.dark.scss

```diff
@@ -0,0 +1,89 @@
+code[class*="language-"], pre[class*="language-"] {
+	font-size: 14px; 
+	line-height: 1.375;
```

Comment:
indentation: one space to much in all file :D

Reply:
we need to setup a linter & formatter. would you mind creating the ticket for it?

Reply:
There we go: https://github.com/deven-org/deven-website/issues/29 <3